pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -56,9 +58,28 @@ def pytorch_funcify_FunctionGraph(
     fgraph_name="pytorch_funcified_fgraph",
     **kwargs,
 ):
+    def conversion_func_register(*args, **kwargs):
+        last_frame = inspect.currentframe().f_back
+        functor = pytorch_funcify(*args, **kwargs)
+        if last_frame:
+            module = sys.modules[last_frame.f_globals["__name__"]]
+            name = functor.__name__
+            i = 0
+            prefix = ""
+            while True:
```

Comment:
This is just placeholder. We need to essentially make sure `elemwise_fn` and `elemwise_fn_1` show up in the guard.